### PR TITLE
Remove fnr from links

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/api/api.ts [hunk 1]
```diff
@@ -70,12 +70,13 @@ export function useFetchAktivEnhet(options?: Options): UseAxiosResponseValue<Akt
 	return useAxios<AktivEnhetResponse>('/modiacontextholder/api/context/aktivenhet', options);
 }
 
-export function useFetchTilgangTilBruker(fnr: string, options?: Options): UseAxiosResponseValue<true> {
-	return useAxios<true>({ url: `/veilarbperson/api/person/${fnr}/tilgangTilBruker` }, options);
+export function useFetchTilgangTilBruker(fnr: string, options?: Options): UseAxiosResponseValue<boolean> {
+	return useAxios<boolean>({ url: `/veilarbperson/api/v2/person/hent-tilgangTilBruker`, method: 'POST', data: {fnr}}, options);
 }
 
+// @todo: check if link is correct!!
```
Comment (Mathiamu) [inline src/api/api.ts:77]: Remember!

File: src/api/api.ts [hunk 2]
```diff
@@ -70,12 +70,13 @@ export function useFetchAktivEnhet(options?: Options): UseAxiosResponseValue<Akt
 	return useAxios<AktivEnhetResponse>('/modiacontextholder/api/context/aktivenhet', options);
 }
 
-export function useFetchTilgangTilBruker(fnr: string, options?: Options): UseAxiosResponseValue<true> {
-	return useAxios<true>({ url: `/veilarbperson/api/person/${fnr}/tilgangTilBruker` }, options);
+export function useFetchTilgangTilBruker(fnr: string, options?: Options): UseAxiosResponseValue<boolean> {
+	return useAxios<boolean>({ url: `/veilarbperson/api/v2/person/hent-tilgangTilBruker`, method: 'POST', data: {fnr}}, options);
```
Comment (dragutin-nav) [inline src/api/api.ts:74]:
```suggestion
	return useAxios<boolean>({ url: `/veilarbperson/api/v3/person/hent-tilgangTilBruker`, method: 'POST', data: {fnr}}, options);
```

File: src/api/api.ts (hunk 1 above)
Comment (dragutin-nav) [inline src/api/api.ts:77]:
```suggestion
```

Comment (dragutin-nav) [reply src/api/api.ts:77]: Checked.

File: src/mock/api/handlers.ts [hunk 3]
```diff
@@ -68,27 +68,27 @@ export const handlers = [
...
-	rest.get('/veilarbperson/api/v2/person/vergeOgFullmakt', responseResolver({ status: 204 })),
+	rest.post('/veilarbperson/api/v3/person/hent-tilgangTilBruker', responseResolver({ json: mockTilgangTilBruker })),
+	rest.post('/veilarbperson/api/v3/person/hent-vergeOgFullmakt', responseResolver({ status: 204 })),
 	rest.get('/veilarbperson/api/person/:fnr/harNivaa4', responseResolver({ json: harniva4MockData })),
-	rest.get('/veilarbperson/api/v2/person', responseResolver({ json: personV2mockData })),
-	rest.get('/veilarbperson/api/v2/person/tolk', responseResolver({ status: 204 })),
+	rest.post('/veilarbperson/api/v3/hent-person', responseResolver({ json: personV2mockData })),
+	rest.post('/veilarbperson/api/v3/person/hent-tolk', responseResolver({ status: 204 })),
 
 	rest.post('/veilarboppfolging/api/v2/manuell/synkroniser-med-dkif', responseResolver({ status: 204 })),
```
Comment (dragutin-nav) [inline src/mock/api/handlers.ts:88]:
```suggestion
	rest.post('/veilarboppfolging/api/v3/manuell/synkroniser-med-dkif', responseResolver({ status: 204 })),
```

File: src/mock/api/handlers.ts [hunk 4]
```diff
@@ -98,7 +98,7 @@ export const handlers = [
 	rest.get('/veilarboppfolging/api/oppfolging/me', responseResolver({ json: meMockData })),
 	rest.get('/veilarboppfolging/api/oppfolging/harFlereAktorIderMedOppfolging', responseResolver({ json: false })),
 	rest.get('/veilarboppfolging/api/oppfolging/mal', responseResolver({ json: malMockData })),
-	rest.post('veilarboppfolging/api/:fnr/lestaktivitetsplan', responseResolver({ status: 204 })),
+	rest.post('veilarboppfolging/api/v2/hent-lestaktivitetsplan', responseResolver({ status: 204 })),
```
Comment (dragutin-nav) [inline src/mock/api/handlers.ts:101]:
```suggestion
	rest.post('veilarboppfolging/api/v3/veileder/lest-aktivitetsplan', responseResolver({ status: 204 })),
```

**Pull request merged**